workspace/apps/pidp/src/app/features/profile/pages/user-access-agreement/user-access-agreement.page.spec.ts

```diff
@@ -61,7 +62,7 @@ describe('UserAccessAgreementPage', () => {
     given('user wants to go back to the previous page', () => {
       when('onBack is invoked', () => {
         component.onBack();
-
+        window.scroll = scroll;
```

Comment:
Just getting info on the console error fix :
below the scrollTop call was added back. Is adding the window.scroll fix the console error ?.

Reply:
I have implemented a fix in the spec.ts file to address the issue. I saved the original window.scroll to a variable and restored it after invoking the onBack method. This change resolves the console error while keeping the scroll functionality intact.

Reply:
The issue was the `UserAccessAgreementPage` test suite was trying to access the `scrollTop()` method that gets invoked in the class but we need to provide the `UtilsService` dependency for the test file. The missing dependency was throwing the `Not Implemented` Error.